tests/perf-bench-numa-mem

```diff
@@ -9,8 +11,15 @@
 . $LKP_SRC/lib/unit.sh
 . $LKP_SRC/lib/reproduce-log.sh
 
+[ -n "$nr_processes" ] || nr_processes=$nr_cpu
+
 mb_proc=$(to_mb $mem_proc)
 
 log_cmd numactl --hard || die "Test needs available numa"
 
-log_cmd perf bench numa mem -p $nr_cpu -t $nr_threads -m -0 -P $mb_proc
+if [ -z "$extra_params" ]
```

Comment:
thanks, seems this check is not needed, if it is empty, the cmd can still be below one
```
log_cmd perf bench numa mem -p $nr_processes -t $nr_threads -m -0 -P $mb_proc $extra_params
```

Reply:
Thanks, updated as per the comment.